test/unit/array_test.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require_relative '../../lib/invoca/utils/array.rb'
+require_relative '../test_helper'
+
+class ArrayTest < Minitest::Test
+  context '* operator' do
+    should 'call the same method on each item in an array and return the results as an array' do
+      assert_equal([4, 5, 5], ['some', 'short', 'words'].*.length)
+    end
```

Comment:
I think you should also test a method with some arguments, like
```
assert_equal(["om", "ho", "or"], ['some', 'short', 'words'].*.slice(1, 2)
```

Reply:
Good idea.